.changeset/curly-bikes-train.md

```diff
@@ -0,0 +1,24 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": feature
+---
+Adds enablable allowed types to SchemaFactoryAlpha
+
+This adds the `enablable` API to [`SchemaFactoryAlpha`](https://fluidframework.com/docs/api/fluid-framework/schemafactoryalpha-class).
+Enablables can be used for schema evolution to add members to an [`AllowedTypes`](https://fluidframework.com/docs/api/fluid-framework/allowedtypes-typealias) while supporting cross version collaboration.
+
+Enablables are allowed types that can be read by the tree but not written. Attempting to write an enablable type will cause a runtime error.
+
+To add a new member to an `AllowedTypes`, add the type wrapped by `enablable`:
```

Comment:
Since the text after this example is discussing this hypothetical upgrade, we should introduce it as a hypothetical upgrade:

```suggestion
To add a new member to an `AllowedTypes`, add the type wrapped by `enablable`.
For example, migrating an array which previously supported only numbers to support both numbers and strings would start by deploying a version of the app using `enablable`:
```